private-templates-service/client/src/components/Common/EditNameModal/styled.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+
+export const EditWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding-bottom: 24px;
+`;
+
+export const StyledInput = styled(TextField)``;
```

Comment:
Do we need `StyledInput` if we're not adding any CSS to it? Can we just use `Textfield`?

Reply:
whats the ``?

Reply:
removed